lib/styles/editor/_menu.scss

```diff
@@ -257,3 +257,18 @@
 .neeto-editor-bubble-menu-animate-shake {
   animation: 1s linear infinite alternate both shake;
 }
+
+.neeto-editor__image-menu {
+  .ne-dropdown__popup {
+    display: flex;
+    min-width: unset;
+  }
+
+  &-btn {
+    position: absolute;
+    top: 0;
+    right: 0;
```

Comment:
Instead of placing it at the extreme corner, a little offset would be better. Try `2px` or `4px`.